Helper/Parser.php

```diff
@@ -104,6 +107,15 @@ public function parseText($text, $checkTable = true) {
 
     $text = str_replace("#", "\\#", $text);
     $text = str_replace("_", "\\_", $text);
+    $text = str_replace("$", "\\$", $text);
+    $text = str_replace("%", "\\%", $text);
+    $text = str_replace("&", "\\&", $text);
+    $text = str_replace("~", "\\~{}", $text);
+    $text = str_replace("^", "\\^{}", $text);
+    $text = str_replace("{", "\\{", $text);
+    $text = str_replace("}", "\\}", $text);
+//    $text = str_replace(">", "\\textlangle{}", $text);
+//    $text = str_replace("<", "\\textrangle{}", $text);
```

Comment:
You mean for "<" and ">"? No, didn't get it to work for those, but would 
like to find out. Workaround would be "$ < $", but creates formula "<" 
which are bigger, it seems to me. "<" in text seems to be an issue in 
general, to be solved with packages, but I could not get it on...

Am 16.06.16 um 14:47 schrieb Bob van de Vijver:
> In Helper/Parser.php
> <https://github.com/bobvandevijver/latex-bundle/pull/19#discussion_r67337426>:
>
>> @@ -104,6 +107,15 @@ public function parseText($text, $checkTable = true) {
>>
>>      $text = str_replace("#", "\\#", $text);
>>      $text = str_replace("_", "\\_", $text);
>> +    $text = str_replace("$", "\\$", $text);
>> +    $text = str_replace("%", "\\%", $text);
>> +    $text = str_replace("&", "\\&", $text);
>> +    $text = str_replace("~", "\\~{}", $text);
>> +    $text = str_replace("^", "\\^{}", $text);
>> +    $text = str_replace("{", "\\{", $text);
>> +    $text = str_replace("}", "\\}", $text);
>> +//    $text = str_replace(">", "\\textlangle{}", $text);
>> +//    $text = str_replace("<", "\\textrangle{}", $text);
>
> Small commit mistake?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bobvandevijver/latex-bundle/pull/19/files/a497336cd71d5313f3a31052b78a5d8d8d943187#r67337426>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEvpCp1ZI9-SYos9YxOJB3sQmARpzmteks5qMUXKgaJpZM4I3HXn>.
>